### ISSUE TITLE
[Bug] NewPipe cannot load certain videos, but can load others

### ISSUE THREAD

**Issue by MatthewR37** [TeamNewPipe/NewPipe#1397]
Hi everyone,

I noticed that NewPipe cannot load some videos, but can load others. The video that doesn't play in NewPipr can be played in YouTube at the same time. This is a recurring issue and happens very often. The videos that it happens to changes often.

**Comment by theScrabi** [comment]
When you stumble accross such a video again post a link here.

**Comment by StarTrekRocks73** [comment]
@theScrabi I can try to, but the video only temporarily doesn't work with NewPipe, meaning that when you investigate it you won't be able to identify the issue. Also the videos that it happens with alternates.

Also, does NewPipe download the videos directly from YouTube.com?

**Comment by theScrabi** [comment]
> Also, does NewPipe download the videos directly from YouTube.com?

 yes

**Comment by gabriellluz** [comment]
It happens to me, too. I'm going to try tonight and I'll send you a log. Does NewPipe has log or something like that? There's a specific channel that it happens a lot.

**Comment by theScrabi** [comment]
we have to bug tracker. if no exception is thrown the only way of getting logs is using a logcat app.

**Comment by eviallet** [comment]
I managed to catch a log of this issue :
![newpipe](https://user-images.githubusercontent.com/29801087/56091159-634e3880-5eab-11e9-973f-8da8f1cdb569.PNG)

After 30 seconds or so, the video starts playing. Here is the full log of : clicking the video, waiting for it to load, and video starting.
![newpipe2](https://user-images.githubusercontent.com/29801087/56091188-ac05f180-5eab-11e9-8445-1cfc9b627425.PNG)

This happens randomly ; however, I can bypass the 30s wait by taping several times on the left of the screen (which trigger going back few seconds earlier, even if the video is not loaded) : after around 10 taps, the video starts.

**Comment by jdramirez165** [comment]
Same problem happened to me. I was attempting to play my favorite video showing the arcade game, but it ends up with just nothing but a loading icon. Wonder whats causing it?

**Comment by Smolhuman** [comment]
I can't play most videos that i regularly watch on newpipe, being music, at first some videos began not loading now the entire playlist except one video plays.
I listen to music in background as a regular human does, not sure if it makes a difference, same videos work fine on regular YouTube.
I get the couldn't load video error, wiping cache sometimes help but still most videos wouldn't play.
Uninstalling the app and downloading the latest version fixed the issue for a hot minute then the problem came back.
It happens mostly in playlists that i play repetitively.
For example the kaiji ost playlist and a playlist i made of battle tendency ost.

**Comment by wvffle** [comment]
Same for me. Some videos used to work and now don't. For example [this song](https://www.youtube.com/watch?v=udkoNdjLzTg) and pretty much for every other video of this song. 
Neither videos nor music in background works. 

Can it be that YouTube is blocking it in some way or they're just changing the API?

**Comment by AlexSD3** [comment]
Uninstalled version 0.16.1 and F-Droid, downloaded 0.16.2 installed and now it works.

**Comment by Stypox** [comment]
This happens sometimes when youtube changes things, so even though this issue has been fixed it could happen again. But it's better to open a new issue every time.
